mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -41,6 +41,60 @@ CON_COMMAND_F(
     pPlayer->TogglePracticeMode();
 }
 
+CON_COMMAND(
+    mom_eyetele,
+    "Teleports the player to the solid that they are looking at.\n")
+{
+    CMomentumPlayer *pPlayer = CMomentumPlayer::GetLocalPlayer();
+    if (!pPlayer)
+        return;
+
+    if (pPlayer->m_bHasPracticeMode || !g_pMomentumTimer->IsRunning())
+    {
+        trace_t tr;
+        Vector pos = pPlayer->EyePosition();;
+        Vector ang;
+        pPlayer->EyeVectors(&ang);
+
+        int mask = CONTENTS_SOLID | CONTENTS_MOVEABLE | CONTENTS_OPAQUE | CONTENTS_WINDOW;
+        UTIL_TraceLine(pos, pos + ang * MAX_COORD_RANGE, mask, pPlayer, COLLISION_GROUP_NONE, &tr);
+
+        if (tr.fraction != 1.0 && tr.DidHit())
+        {
+            Vector hit = tr.endpos;
+            if (enginetrace->PointOutsideWorld(hit))
+            {
+                hit += (hit - pos).Normalized() * 64;
```

Comment:
Since you end up reusing this `(hit - pos).Normalized()` value below as well, store it in a variable, preferably declared under `hit`.

Use 64.0f instead of an integer here.

Reply:
hit could end up being modified, so it wouldn't be the same for both instances.